lnbits/wallets/lndgrpc.py

```diff
@@ -144,8 +144,12 @@ async def create_invoice(
         payment_request = str(resp.payment_request)
         return InvoiceResponse(True, checking_id, payment_request, None)
 
-    async def pay_invoice(self, bolt11: str) -> PaymentResponse:
-        resp = self.rpc.send_payment(payment_request=bolt11)
+    # WARNING: correct handling of fee_limit_msat is required to avoid security vulnerabilities!
+    # The backend MUST NOT spend satoshis above invoice amount + fee_limit_msat.
+    async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
+        fee_limit_fixed = ln.FeeLimit(fixed=fee_limit_msat//1000)
```

Comment:
Should this be fixed=fee_limit_msat/1000

Reply:
Actually, LND supports msats: https://api.lightning.community/#lnrpc-feelimit
So `fee_limit_fixed = ln.FeeLimit(fixed_msat=fee_limit_msat)` should work.

Reply:
I am afraid [the library](https://github.com/adrienemery/lnd-grpc-client/blob/master/lndgrpc/rpc_pb2.py#L501) is not ready for this. It needs to be a satoshi integer... But please correct me if I am being wrong.

Reply:
Looks like that library is not updated. So I'd suggest use integer now and try to look for a better maintained library.

Reply:
Yes, needs integer, should be `//`.

Reply:
> 

We're already switching to another library (see `fastapi` branch).